data/calculated_parser/lexer.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, **kwargs):
                 'TIMES',
                 'DIVIDE',
                 'POWER',
+                'TILDA',
```

Comment:
I think the correct spelling here is "tilde" (yeah English, as usual haha).

Reply:
Ohhh... I guess i've been spelling it wrong all my life

Reply:
Better late than never 🤣 https://en.wikipedia.org/wiki/Tilde

Reply:
fixed